cmake/Dependencies.cmake

```diff
@@ -1699,9 +1699,6 @@ if(NOT INTERN_BUILD_MOBILE)
     add_compile_options(-DUSE_GCC_GET_CPUID)
   endif()
 
-  find_package(VSX) # checks VSX
-  find_package(AVX) # checks AVX and AVX2
```

Comment:
You can't remove those, as lines 1708 and 1712 are conditional on the fact that AVX/AVX2 is found

Reply:
These lines are moved to `cmake/MiscCheck.cmake` and `cmake/Dependencies.cmake` is included after `cmake/MiscCheck.cmake`: https://github.com/pytorch/pytorch/blob/7ac8054d5a10ebe917d3570e325281264ec7b5b3/CMakeLists.txt#L621-L632

Reply:
I agree this should be okay. If for no other reason than `CAFFE2_COMPILER_SUPPORTS_AVX512_EXTENSIONS` is used inside `Dependencies.cmake` so the dependency on `MiscCheck.cmake`  already existed.

Reply:
Calling `find_package(AVX)` is fast as results are cached, if called multiple types. Relying on particular include order of .cmake files is prone to failures.

Reply:
I've added it back. Just to make sure we are on the same page, we added it back solely for the purpose of code maintainability. Functionally it doesn't change anything.